docs/user/howto_gateway_api.md

```diff
@@ -0,0 +1,411 @@
+serviceentries                      se                  networking.istio.io/v1              true         ServiceEntry
+sidecars                                                networking.istio.io/v1              true         Sidecar
+virtualservices                     vs                  networking.istio.io/v1              true         VirtualService
+workloadentries                     we                  networking.istio.io/v1              true         WorkloadEntry
+workloadgroups                      wg                  networking.istio.io/v1              true         WorkloadGroup
+istiocnis                                               sailoperator.io/v1alpha1            false        IstioCNI
+istiorevisions                      istiorev            sailoperator.io/v1alpha1            false        IstioRevision
+istios                                                  sailoperator.io/v1alpha1            false        Istio
+remoteistios                                            sailoperator.io/v1alpha1            false        RemoteIstio
+authorizationpolicies               ap                  security.istio.io/v1                true         AuthorizationPolicy
+peerauthentications                 pa                  security.istio.io/v1                true         PeerAuthentication
+requestauthentications              ra                  security.istio.io/v1                true         RequestAuthentication
+telemetries                         telemetry           telemetry.istio.io/v1               true         Telemetry
+
+# A gateway class for applications to use gateways on
+$ oc get gatewayclass openshift-gateway-api -o yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: GatewayClass
+metadata:
+  creationTimestamp: "2024-11-05T13:15:50Z"
+  generation: 1
+  name: openshift-gateway-api
+  resourceVersion: "1518"
+  uid: 2490e78e-9000-42cf-8234-d3e5482825a9
+spec:
+  controllerName: openshift.io/gateway-controller
+status:
+  conditions:
+  - lastTransitionTime: "2024-11-05T13:18:02Z"
+    message: Handled by Istio controller
+    observedGeneration: 1
+    reason: Accepted
+    status: "True"
+    type: Accepted
+
+# Service mesh operator and istio controller
+$ oc get deployment -n openshift-gateway-api
+NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+istiod-openshift-gateway-api   1/1     1            1           26h
+servicemesh-operator3          1/1     1            1           26h
+```
+These resources are handled by MicroShift and will always be present.
+
+## Configuring Gateways
+In order to use the feature we only need an application that we want to expose through a gateway, a gateway, and some routes.
+
+First we create a sample application:
+```bash
+$ oc create -f microshift/test/assets/hello-microshift.yaml 
+pod/hello-microshift created
+
+$ oc create -f microshift/test/assets/hello-microshift-service.yaml 
+service/hello-microshift created
```

Comment:
I'm not sure we need outputs for the creation commands. Let's just give the commands and check the pod/svc output?